packages/fxa-auth-server/test/local/payments/capability.js

```diff
@@ -241,6 +247,74 @@ describe('CapabilityService', () => {
     });
   });
 
+  describe('fetchSubscribedPricesFromAppStore', () => {
```

Comment:
I didn’t see any existing tests for `subscribedPriceIds` or `fetchSubscribedPricesFromPlay`, but I added some tests for `fetchSubscribedPricesFromAppStore`.

Reply:
Per Ben, we should have tests here. I will add one for `fetchSubscribedPricesFromPlay` since it's so similar to `fetchSubscribedPricesFromAppStore`.